ci/ci.sh

```diff
@@ -76,6 +76,9 @@ function run_tests {
     time HYPERFINE="hyperfine -w0 -r1" ./miri bench $TARGET_FLAG
   fi
 
+  ## Test that `miri run` with dependencies works
+  ./miri run $TARGET_FLAG --dep tests/pass-dep/getrandom.rs
```

Comment:
I'm going to move this in its own PR, it shouldn't be blocked on the ui_test update.